pymc3/distributions/multivariate.py

```diff
@@ -2036,30 +2080,34 @@ def logp(self, value):
         TensorVariable
         """
 
+        D = W.sum(axis=0)
+
+        Dinv_sqrt = np.diag(1 / np.sqrt(D))
+        DWD = np.matmul(np.matmul(Dinv_sqrt, W), Dinv_sqrt)
+        lam = scipy.linalg.eigvalsh(DWD)
```

Comment:
NumPy can't be used here; all these variables are symbolic (i.e. `TensorVariable`s).  You'll need to use their Aesara equivalents.